### PR TITLE
edits to the CMFA definition

### PR DESCRIPTION
**Pull request by woodbe**
The original definition was pulled directly from the ISO description, but the statement of "computer" seems a little specific (while they are, smartphones aren't usually called computers), and it would also seem to preclude things like a wearable that is associated with the phone (as sensor isn't necessarily the right point of connection for a sensor on a wearable that is paired to a phone that is then used for CMFA).

### CODE REVIEW

File: CMFA_ESR.adoc [hunk 1]
```diff
@@ -42,7 +42,7 @@ This section needs to be reviewed
 ====
 The {itc-longname} iTC has been requested by vendors to develop an Essential Security Requirements (ESR) for Continuous Multifactor Authentication (CMFA) products. The CMFA ESR version 1.0 contains material generated by the members of the {itc-shortname}, for a PP-Module to be reviewed by the membership of the {itc-shortname} and endorsed in 2021.
 
-CMFA is an implicit process of validating the user periodically to grant access to the legitimate user and to deny access to impostors based on captured biometric characteristics and built-in sensors of the computer device.
+CMFA is an implicit process of validating the user periodically to grant access to the legitimate user and to deny access to impostors based on captured biometric characteristics and built-in sensors of the device (or connected devices).
```
Comment (gfiumara) [inline CMFA_ESR.adoc]: Suggest making the parenthetical part of the sentence and making this _and_ not _or_, to not imply one or the other.
```suggestion
CMFA is an implicit process of validating the user periodically to grant access to the legitimate user and to deny access to impostors based on captured biometric characteristics and built-in sensors of the device and connected devices.
```

**Review by gfiumara — APPROVED** [review]

**Comment by gregott** [comment]
Looks good to me.

**Review by gregott — APPROVED** [review]

**Comment by woodbe** [comment]
Mark this to be reviewed in #24 by @woodbe

**Comment by woodbe** [comment]
Closing as it looks like this is sufficiently covered in the update

**Pull request closed without merging**